modules/navier_stokes/test/tests/finite_volume/ins/block_restriction/2d-rc.i

```diff
@@ -252,9 +252,8 @@ restricted_blocks = '1'
 [Executioner]
   type = Steady
   solve_type = 'NEWTON'
-  petsc_options_iname = '-pc_type -ksp_gmres_restart -sub_pc_type -sub_pc_factor_shift_type'
-  petsc_options_value = 'asm      100                lu           NONZERO'
-  line_search = 'none'
+  petsc_options_iname = '-pc_type -pc_factor_shift_type'
+  petsc_options_value = 'lu NONZERO'
```

Comment:
With only two options this is fine as-is, but in general I like to space the values to align with the names to help readability.

Reply:
Got it, will pay attention to it!